cmd/podman/common.go

```diff
@@ -197,6 +197,11 @@ var createFlags = []cli.Flag{
 		Name:  "hostname",
 		Usage: "Set container hostname",
 	},
+	cli.StringFlag{
+		Name:  "image-volume, builtin-volume",
```

Comment:
No I was fine with the flag, I was just looking for a better help message stating that image-volume told podman how to handle building volumes in an image.

Reply:
reworded a bit.